cws-service/src/main/java/jpl/cws/process/initiation/aws/S3DataManager.java

```diff
@@ -26,25 +26,30 @@
  * @author ghollins, jwood, ztaylor
  */
 public class S3DataManager {
-		private static final Logger log = LoggerFactory.getLogger(S3DataManager.class);
+	private static final Logger log = LoggerFactory.getLogger(S3DataManager.class);
 
-		public static final int OLD_SECONDS_BACK_THRESHOLD = 120;
+	public static final int OLD_SECONDS_BACK_THRESHOLD = 120;
 
-		private Region regionUsed;
+	private Region regionUsed;
 
-		private S3Client s3;
+	private S3Client s3;
 
-		public S3DataManager(String region) {
-			init(region); // sets defaults
-		}
+	public S3DataManager(String region) {
+		init(region); // sets defaults
+	}
 
-		public void init(String region) {
-			s3 = S3Client.builder().region(Region.of(region)).build();
-			regionUsed = Region.of(region);
-		}
+	public void init(String region) {
+		s3 = S3Client.builder().region(Region.of(region)).build();
+		regionUsed = Region.of(region);
+	}
 
+	public void close() {
```

Comment:
```suggestion
	@Override 
	public void close() throws Exception {
```
Add override annotation for AutoClosable interface